packages/compiler/src/types.ts

```diff
@@ -91,13 +94,51 @@ export interface ImportExpression {
   reference?: string;
 }
 
+interface IWithBlock {
+  [BLOCK_HEIGHT_KEY]: number;
+}
+
+interface ISourceWithBlockHeight extends IWithBlock {
+  "": string;
+}
+
```

Comment:
Super duper nit picky, but we can probably rename these to `WithBlock` and `SourceWithBlockHeight`. Beginning interfaces with `I` (like in C Sharp) isn't very typical in TypeScript from my experience. Feel free to ignore this if anyone strongly disagrees though.

Reply:
Nice catch! I've updated interface names to be consistent.